src/Application.js

```diff
@@ -55,11 +56,11 @@ export class Application {
 
     // listen for events from the container and keep the local value in sync
     [
-      'soundMuted',
+      'soundVolume',    
```

Comment:
We have these strings repeating a lot in this file. We should store them in variables to cut down some redundancy.